src/swagger/openapi.json

```diff
@@ -0,0 +1,305 @@
+          },
+          "401": {
+            "description": "Invalid user session",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/articles": {
+      "post": {
+        "tags": [
+          "articles"
+        ],
+        "summary": "Create an article",
+        "operationId": "createArticle",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "activeUserId",
+            "schema": {
+              "type": "string",
+              "format": "uuid"
+            },
+            "required": true
+          }
+        ],
+        "requestBody": {
+          "description": "Created article object",
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "content": {
+                    "type": "string"
+                  },
+                  "visibility": {
+                    "type": "string"
+                  },
+                  "activeUserId": {
+                    "type": "string"
+                  },
+                  "title": {
+                    "type": "string"
+                  },
+                  "article_id": {
+                    "type": "string"
+                  }
+                }
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "201": {
+            "description": "Article created"
+          },
+          "400": {
+            "description": "Invalid visibility parameter",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Invalid token",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "articles"
+        ],
+        "summary": "Get articles",
+        "operationId": "getArticles",
+        "responses": {
+          "200": {
+            "description": "Articles retrieved",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "content": {
+                        "type": "string"
+                      },
+                      "visibility": {
+                        "type": "string"
+                      },
+                      "activeUserId": {
```

Comment:
[GET /api/articles](https://github.com/waterseeker/js-rest-api/blob/a6d1187d833de000fcfd18a0b9edca83bab7c9d0/src/routes/articles.js#L5) returns an array of articles that are shaped as the [Article model](https://github.com/waterseeker/js-rest-api/blob/a6d1187d833de000fcfd18a0b9edca83bab7c9d0/src/models/Article.js). That means that [these](https://github.com/waterseeker/js-rest-api/blob/a6d1187d833de000fcfd18a0b9edca83bab7c9d0/src/models/Article.js#L3-L11) are the field names. `activeUserId` needs to be changed to `author`.

Reply:
Thanks. That made me realize I had them listed in a different order in the docs too. Small thing but I think it's better if the order matches.